erpnext/accounts/doctype/journal_entry/journal_entry.js

```diff
@@ -451,10 +451,20 @@ frappe.ui.form.on("Journal Entry Account", {
 	},
 
 	debit_in_account_currency: function(frm, cdt, cdn) {
+		var row = locals[cdt][cdn];
+		if(row.debit_in_account_currency < 0 || row.debit_in_account_currency === 0){
+			frappe.model.set_value(cdt, cdn, "debit_in_account_currency", '');
```

Comment:
why you are setting it as blank?

Reply:
On considering, Journal Entry can't be created with 0 credit/debit amount. Hence set it to blank.